src/test/resources/testResult/featureCollection/ANC/featureCollection_bbox.json

```diff
@@ -48,6 +48,7 @@
         "hydrologicUnit": null,
         "state": null,
         "nationalAquifer": null,
+                "nationalAquiferCode": null,
```

Comment:
whitespace?

Reply:
Yes, this is one of the reasons spaces are preferred by some (including me). The tab char in the IDE has different tab sizing than most other tools. It will probably align fine in the IDE. It is odd; our IDEs default to a 4 space tab while all other tools default to an 8 space tab. If we just use spaces then it would always be aligned.

Reply:
Yeah, I prefer spaces as well.